src/component/App.js

```diff
@@ -1,9 +1,28 @@
 import React from "react";
+import {
+  Provider,
+  Context,
+  ErrorBoundary,
+  useRollbar,
+  useRollbarCaptureEvent,
+  LEVEL_INFO,
+  useRollbarPerson,
+  useContext,
+  RollbarContext,
+  historyContext
+} from '@rollbar/react';
```

Comment:
I understand you may have done this to see what's there or what to use or for effect however, there are 2 better ways to handle this:

1. The appropriate way that React and JS devs will see would be to only import those items that you are going to use in your file, in this case: `Provider`
2. The other way is to use `*` to import everything in an alias, e.g.: `import * as rollbarReact from '@rollbar/react';` and use it as `<rollbarReact.Provider>`

Reply:
Sweet, thanks for the suggestions. I will probably just use the first one so it is clear what part of the library is being used